core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -297,8 +307,9 @@ private static class DefaultClientRequestMetrics
         @Nullable
         private DistributionSummary failureAttempts;
 
-        DefaultClientRequestMetrics(MeterRegistry parent, MeterIdPrefix idPrefix) {
-            super(parent, idPrefix);
+        DefaultClientRequestMetrics(MeterRegistry parent, MeterIdPrefix idPrefix,
+                                    DistributionStatisticConfig distributionStatisticConfig) {
```

Comment:
Shouldn't we also this `distributionStatisticConfig` for timers/distributions in this class?

https://github.com/line/armeria/blob/337c09cb2b62a1438c3195e858e82bf89f85dc20/core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java#L316-L323

https://github.com/line/armeria/blob/337c09cb2b62a1438c3195e858e82bf89f85dc20/core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java#L369-L386